.github/workflows/instrumentation-matrix-nightly.yaml

```
@@ -0,0 +1,313 @@
+        with:
+          name: nightly-summary
+          path: test/instrumentation-matrix/reports/summary.md
+
+  open-issue-on-failure:
+    runs-on: ubuntu-latest
+    needs: publish-matrix-summary
+    # `needs.heavy-tier.result == 'failure'` is a no-op today because heavy
+    # is continue-on-error; left in so the condition is correct the moment
+    # the heavy bodies land and that flag flips.
+    if: |
+      needs.publish-matrix-summary.outputs.has_failures == 'true' ||
```

actionlint (1.7.12)

[error] property "full-emission-matrix" is not defined in object type {publish-matrix-summary: {outputs: {counts_json: string; has_failures: string}; result: string}}

(expression)